src/main/java/com/feedzai/commons/sql/abstraction/engine/DatabaseEngineUniqueConstraintViolationException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.feedzai.commons.sql.abstraction.engine;
```

Comment:
To avoid having lots of stuff in the same place, please move this to
`com.feedzai.commons.sql.abstraction.exceptions`